### ISSUE TITLE
Add Normalization functions to juliamind.nn.functional.norm

### ISSUE THREAD

**Issue by diksha-shrivastava13** [diksha-shrivastava13/JuliaMind#9]
> Please create a new issue with each of the subtasks and comment "- [ ] #issue_number" to keep the task list updated.

- [ ] normalize
- [ ] batch_norm
- [ ] layer_norm
- [ ] instance_norm
- [ ] local_response_norm
